source/core/src/main/com/csse3200/game/components/combat/CombatActions.java

```diff
@@ -50,6 +73,7 @@ private void onCombatWin(Screen screen, ServiceContainer container) {
     // Set current screen to original MainGameScreen
 //    game.setOldScreen(screen, container);
     game.setScreen(GdxGame.ScreenType.GAME_OVER_WIN);
+    entity.getEvents().trigger("onCombatWin", manager.getPlayerStats());
```

Comment:
We should be right to get rid of the commented out code above here now (sorry it was from last pull request)